pkg/activator/util/transports_test.go

```diff
@@ -61,13 +61,10 @@ func TestHTTPRoundTripper(t *testing.T) {
 
 	for _, e := range examples {
 		t.Run(e.label, func(t *testing.T) {
-			wants[e.want] = false
```

Comment:
Should we keep this as `wants.Delete(e.want)` here?  It makes it so that we know *this* call to `RoundTrip` populates the entry in `wants` IIUC.